packages/babel-traverse/test/fixtures/conversion/implicit-var-arguments/options.json

```diff
@@ -0,0 +1,4 @@
+{
+  "sourceType": "unambiguous",
+  "presets": ["env"]
```

Comment:
The Babel 8 tests are failing because you didn't specify any target here. Could you move this test in the arrow functions plugin, and enable the plugin instead of the preset?
Also, please specify `"sourceType": "script"` since this is specific to loose mode.

Reply:
Okay, I can do that.

Reply:
I changed the same as you said.